docs/global-styles/prism.ts

```diff
@@ -14,7 +14,7 @@ export const globalPrism = css`
     position: relative;
     padding: 0.2em;
     border-radius: 0.3em;
-    color: color: ${theme.palette.red11};
+    color: ${theme.palette.red11};
```

Comment:
```suggestion
```

This styling is no longer intended, and after fixing it leads to unwanted side-effects. 😅